package_control/commands/existing_packages_command.py

```diff
@@ -62,7 +62,10 @@ def make_package_list(self, action=''):
                 description = '<em>%s</em>' % sublime.html_format_command(description)
                 final_line = '<em>' + action + installed_version + '</em>'
                 if url_display:
-                    final_line += '; <a href="%s">%s</a>' % (url, url_display)
+                    final_line += '; <a href="%s">%s</a>' % (
+                        sublime.html_format_command(url),
+                        sublime.html_format_command(url_display),
```

Comment:
I don't think using `html_format_command` is appropriate since it's not a command that we are formatting. We just want to encode chars as entities.

Looking at the implementation of `html_format_command`, it should be enough to use `html.escape` instead.

```py
def html_format_command(cmd, args=None):
    return html.escape(format_command(cmd, args), quote=True)
```

(I guess it's not really specific to your change as I see that it was used above already but IMO it's not really necessary or appropriate.)